package.json

```diff
@@ -60,12 +60,12 @@
     "react": "16.3.1",
     "react-dom": "16.2.0",
     "react-native": "0.55.4",
-    "react-native-vector-icons": "^4.4.2",
+    "react-native-vector-icons": "^5.0.0",
     "react-test-renderer": "16.3.1",
     "webpack": "^2.2.1"
   },
   "peerDependencies": {
-    "react-native-vector-icons": "^4.2.0 || ^5.0.0"
+    "react-native-vector-icons": "^5.0.0"
```

Comment:
Why you disallow v4 usage?

Reply:
Because it'll break if version below 5 is used. FontAwesome5 is only in version 5. So the import will break the app

Reply:
Then imo You should replace current font awesome with version 5 in code instead adding it as new font provider. With this change You can't install locally fontawesome4 and use it.

Reply:
That's not true, react-native-vector-icons v5 supports both font-awesome@4^ and 5. https://github.com/oblador/react-native-vector-icons#bundled-icon-sets. 

You'll only need to update the react-native-vector-icons package when you update react-native-elements.